src/Infrastructure/Persistence/OMDB/MovieRepository.php

```diff
@@ -8,15 +8,27 @@
 class MovieRepository implements Movie\MovieRepository
 {
     protected $adapter;
+    protected $movieBuilder;
 
     public function __construct(Adapter\Adapter $adapter)
     {
         $this->adapter = $adapter;
+        $this->movieBuilder = new Movie\MovieBuilder();
```

Comment:
I don't normally like creating new objects inside constructors but this one is simple enough that I went with it.